client/reader/list-stream/empty.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import EmptyContent from 'calypso/components/empty-content';
+import getPreviousRoute from 'calypso/state/selectors/get-previous-route';
+
+export default function ListEmptyContent(): JSX.Element {
+	const translate = useTranslate();
+	const previousRoute: string = useSelector( getPreviousRoute );
+
+	function previousRouteIsUserProfileLists(): boolean {
+		return /^\/read\/users\/[a-z0-9]+\/lists$/.test( previousRoute );
```

Comment:
I think this is where it is failing now. It seems the `getPreviousRoute` is adding a `?` to the end of the string regardless of presence of query params, which makes this always pop `false`.